src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>5</number>
```

Comment:
:information_source: Try to remember to select the first tab before saving the dialogue/form - though there is now code in place to force the first tab to be shown (except in a particular special case) this change happens a lot because people forget it and it gets logged into the git history when ideally it shouldn't be...